README.md

```diff
@@ -37,10 +37,12 @@ Since webshims does not support fingerprinting, this will result in 404s (missin
    (Note that this should be run directly, not in a dom-ready block.)
 
   ```javascript
-  $.webshims.setOptions('basePath', '/assets/webshims/shims/')
+  $.webshims.setOptions('basePath', '/webshims/shims/1.15.6/')
```

Comment:
I agree, which is why I created the `webshims_path` helper. However, in our case (and this seemed likely for most implementations), this line is in a JS file that's loaded and compressed with the rest of the JS into `application.js` and therefore doesn't run through ERB (or HAML or whatever). 

The problem I've found, running this branch, is that the webshims are no longer offloaded to nginx and so they use the rails server which they really shouldn't need to. 

I think I like your approach of just writing webshims to public and letting the assets manager (nginx, CDN, etc.) take care of it from there.